UnityProject/Assets/Scripts/Chemistry/ReactionLib.cs

```diff
@@ -9,27 +9,18 @@
 
 namespace Chemistry
 {
-
-    internal class MainClass
-    {
-        public static void Main(string[] args)
-        {
-            Initialization.run();
-            //Demonstration code//
-            var area = new Dictionary<String, float>();
-            area.Add("potassium", 59.0f);
-            area.Add("oxygen", 49.0f);
-            area.Add("sugar", 45.0f);
-            area.Add("iodine", 20.0f);
+            //Initialization.run();
+            //var area = new Dictionary<String, float>();
+            //area.Add("potassium", 59.0f);
+            //area.Add("oxygen", 49.0f);
+            //area.Add("sugar", 45.0f);
+            //area.Add("iodine", 20.0f);
 
             //area.Add("virusfood",59f);
             //area.Add("blood",49f);
-            float Temperature = 400f;
-            Dictionary<string, float> area_new = Calculations.Reactions(area,Temperature);
+            //float Temperature = 400f;
+            //Dictionary<string, float> area_new = Calculations.Reactions(area,Temperature);
 
-            Console.WriteLine("Press any key to continue . . . ");
-            Console.ReadKey();
-        }
         public class Reaction
```

Comment:
Would you be able to add some Summary comments to the fields in the reaction class so people can get an idea of how to use it. Summary comments also help with VS tooltips